NMS-13324: Store service monitor status in RRD (#3439)
OpenNMS/opennms: docs/modules/operation/pages/service-assurance/polling-packages.adoc

IMPORTANT: If you change the polling interval afterwards, you must recreate existing RRD files with the new definitions.

[[ga-pollerd-packages-service-status-persistence]]
== Service Status Persistence

Each Service Monitor can be configured to store the current status of the Service in RRD files.
This allows to query the current status and the status history similar to response times.

This feature is disabled by default and must be enabled for each service individually.

> To do so, a service parameter must be added to the according services in `poller-configuration.xml`:

Reviewer: ```suggestion
To do so, add a service parameter to the appropriate services in `poller-configuration.xml`:
```
